src/index.html

```diff
@@ -4,15 +4,75 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, initial-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute is linking to a SCSS file. Typically, you should link to a compiled CSS file in HTML, not SCSS. Ensure that the SCSS is compiled to CSS and link to the resulting CSS file instead.